lib/live/page.rb

```diff
@@ -99,7 +99,7 @@ def run(connection)
 						Console.logger.warn(self, "Could not resolve element:", message)
 					end
 				elsif id = message[:id]
-					self.handle(id, message[:event])
+					self.handle(id, message[:event], message[:event][:details])
```

Comment:
Is this necessary? I believe the `message[:event]` can be decoded by the `handle` method.

Reply:
printing the incoming message gives `{:id=>"hform", :event=>{:type=>"submit", :details=>"[\"none\",[[\"a\",\"abc\"],[\"b\",\"dewfg\"]]]"}}`
also, in the original [forward](https://github.com/socketry/live-js/blob/4a0fdac8d6c78b14879504b10f357b1d0656fd1b/src/index.js#L94) method, it is enclosed in the `:event`.

Reply:
[56b91f0](https://github.com/socketry/live/pull/6/commits/56b91f032dd7b9c01354d0978ef24a849f6e0381)